presto-main/src/main/java/io/prestosql/testing/TestingAccessControlManager.java

```diff
@@ -285,6 +288,17 @@ public void checkCanCreateViewWithSelectFromColumns(SecurityContext context, Qua
         }
     }
 
+    @Override
+    public void checkCanShowColumnsMetadata(SecurityContext context, CatalogSchemaTableName table)
+    {
+        if (shouldDenyPrivilege(context.getIdentity().getUser(), table.getSchemaTableName().getTableName(), SELECT_TABLE)) {
```

Comment:
this should be SHOW_COLUMNS, not SELECT_TABLE. It could be possible that user can see table metadata but being not able to select from it.

Reply:
sure